<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-G0D562V" ignored="0" name="Test" tests="5" failures="1" timestamp="2020-06-03T21:29:24 IST" time="119.658" errors="0">
  <testcase name="@AfterMethod afterMethod" time="1.324" classname="AP_TestCases.AP_RegressionTest">
    <failure type="java.lang.AssertionError" message="Failed with addFailLog method caught in after test">
      <![CDATA[java.lang.AssertionError: Failed with addFailLog method caught in after test
at Utilities.ElementUtil.addFailLog(ElementUtil.java:402)
at Utilities.ElementUtil.afterMethod(ElementUtil.java:462)
at AP_TestCases.AP_RegressionTest.afterMethod(AP_RegressionTest.java:40)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@BeforeMethod BeforeMethod" time="-1.591199964643E9" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod BeforeMethod -->
  <testcase name="@AfterMethod afterMethod" time="-1.591199964643E9" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@BeforeMethod BeforeMethod" time="-1.591199964643E9" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod BeforeMethod -->
  <testcase name="@AfterMethod afterMethod" time="-1.591199964659E9" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@BeforeMethod BeforeMethod" time="-1.591199964659E9" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod BeforeMethod -->
  <testcase name="@AfterMethod afterMethod" time="-1.591199964659E9" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="AT_01_NewAccount" time="37.246" classname="AP_TestCases.AP_RegressionTest"/>
  <testcase name="AT_01_NewAccount" time="21.105" classname="AP_TestCases.AP_RegressionTest">
    <failure type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for visibility of element located by By.linkText: Sign in (tried for 20 second(s) with 500 milliseconds interval)">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.linkText: Sign in (tried for 20 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at Utilities.ElementUtil.waitForElementVisibilityByLocator(ElementUtil.java:70)
at AP_Pages.AP_HomePage.verifyAP_HomePageVisible(AP_HomePage.java:18)
at AP_TestCases.AP_RegressionTest.AT_01_NewAccount(AP_RegressionTest.java:48)
at java.util.ArrayList.forEach(ArrayList.java:1257)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"link text","selector":"Sign in"}
  (Session info: chrome=83.0.4103.61)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-G0D562V', ip: '192.168.0.8', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_211'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 83.0.4103.61, chrome: {chromedriverVersion: 83.0.4103.39 (ccbf011cb2d2b..., userDataDir: C:\Users\sunshine\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:52001}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 20030f95eab9cdba4b37726c4836559e
*** Element info: {Using=link text, value=Sign in}
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByLinkText(RemoteWebDriver.java:380)
at org.openqa.selenium.By$ByLinkText.findElement(By.java:220)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:205)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:201)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
... 31 more
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AT_01_NewAccount -->
  <testcase name="AT_01_NewAccount" time="0.0" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- AT_01_NewAccount -->
  <testcase name="AT_01_NewAccount" time="0.0" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- AT_01_NewAccount -->
  <testcase name="AT_01_NewAccount" time="0.0" classname="AP_TestCases.AP_RegressionTest">
    <skipped/>
  </testcase> <!-- AT_01_NewAccount -->
</testsuite> <!-- Test -->
